### ISSUE TITLE
HTML Redirects Missing

### ISSUE THREAD

**Issue by danielzgtg** [rust-lang/rust-forge#336]
https://github.com/rust-lang/rust/blob/7f3df5772439eee1c512ed2eb540beef1124d236/RELEASES.md#compiler-1 links to https://forge.rust-lang.org/platform-support.html but it is now a 404.

I had to dig around to find out that it has been moved to https://forge.rust-lang.org/release/platform-support.html now. Looking through [archive.org](https://web.archive.org/web/20190301000000*/https://forge.rust-lang.org/platform-support.html), it seems that the page was indeed moved and there used to be a redirect to the proper page. However, that redirect is missing now. It would be nice for the redirect to be restored or the dead link to be fixed.

**Comment by XAMPPRocky** [comment]
Thank you for your issue! The code for this here. I can replicate that its not generating redirects, however I can't tell why. It doesn't panic, and seems to be writing to the correct paths.

https://github.com/rust-lang/rust-forge/blob/2c1abeeb7552c6ea0e5330a169094729a316de1c/blacksmith/src/lib.rs#L173-L183

**Comment by danielzgtg** [comment]
I tried to do a Git bisect but ran into a strange problem. I can't find the last good commit.

I've tried d89a96a which is the first commit with mdbook, edd938e which is an early commit with blacksmith, 8fcbfe7 which is between 2019-09-21 and 2019-10-08, as well as many other commits. In all of those cases, I get a 404 when visiting http://127.0.0.1:3000/platform-support.html after running `mdbook serve`. It doesn't generate a file in `book/` either.

This is unexpected because [archive.org](https://web.archive.org/web/20190921031341/https://forge.rust-lang.org/platform-support.html) clearly shows that the redirects were present at some time. This fact worsens the problem because I can't reproduce that desired behavior.
